app/controllers/tools_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ToolsController < ApplicationController
```

Comment:
yeah, this making tools plural and not singular in the filename tripped me up a couple times.